app/konnect/runtime-manager/consumer-groups.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Create Consumer Groups in Konnect
+content_type: tutorial
+badge: enterprise
+---
+With consumer groups, you can define any number of rate limiting tiers and
+apply them to subsets of consumers, instead of managing each consumer
+individually.
+
+For example, you could define three consumer groups:
+* **gold tier** with 1000 requests per minute
+* **silver tier** with 10 requests per second
+* **bronze tier** with 6 requests per 30 seconds
+
+The `consumer_groups` endpoint works together with the [Rate Limiting Advanced plugin](/hub/kong-inc/rate-limiting-advanced).
+
+Consumers that are not in a consumer group default to the Rate Limiting advanced
+plugin’s configuration, so you can define tier groups for some users and
+have a default behavior for consumers without groups.
+
+To use consumer groups for rate limiting, you must:
+* Create a consumer group and create or add consumers to it
+* Configure the Rate Limiting Advanced plugin globally with the `enforce_consumer_groups`
+and `consumer_groups` parameters, setting up the list of consumer groups that
+the plugin accepts
+* Configure a rate limiting policy for each consumer group, overriding the 
+plugin's global configuration
+
```

Comment:
If you introduce this like
"This tutorial will walk through creating a bronze tier consumer group...:"